### ISSUE TITLE
I'm confused about this code“class Normalization(tf.keras.layers.experimental.preprocessing.PreprocessingLayer):”

### ISSUE THREAD

**Issue by Wang-Dlut** [markub3327/Solar-Transformer#6]
I'm confused about this code“class Normalization(tf.keras.layers.experimental.preprocessing.PreprocessingLayer):”，It makes me feel that this code is very complex

**Comment by markub3327** [comment]
@Wang-Dlut Only for backward compatibility with older versions of Keras.

**Comment by Wang-Dlut** [comment]
Thank you very much！ There is important news in the email, hahaha
